docs/en/engines/table-engines/mergetree-family/mergetree.md

```diff
@@ -1248,8 +1247,6 @@ Limit parameters (mainly for internal usage):
 
 Other parameters:
 * `metadata_path` - Path on local FS to store metadata files for Blob Storage. Default value is `/var/lib/clickhouse/disks/<disk_name>/`.
-* `cache_enabled` - Allows to cache mark and index files on local FS. Default value is `true`.
```

Comment:
It'd be better to delete it in a more consistent way
```
─ rg -aF 'cache_enabled' .
./docs/en/operations/storing-data.md
163:                <data_cache_enabled>1</data_cache_enabled>

./programs/diagnostics/testdata/configs/xml/config.xml
1212:                <cache_enabled>true</cache_enabled>

./docs/zh/engines/table-engines/mergetree-family/mergetree.md
748:            <cache_enabled>true</cache_enabled>
775:- `cache_enabled` - 是否允许缓存标记和索引文件。默认值为`true`。

./docs/en/engines/table-engines/mergetree-family/mergetree.md
1223:            <cache_enabled>true</cache_enabled>
1251:* `cache_enabled` - Allows to cache mark and index files on local FS. Default value is `true`.

./docs/ru/engines/table-engines/mergetree-family/mergetree.md
808:            <cache_enabled>true</cache_enabled>
835:-   `cache_enabled` — признак, разрешено ли хранение кэша засечек и файлов индекса в локальной файловой системе. Значение по умолчанию: `true`.

./programs/server/config.xml
451:                <cache_enabled>true</cache_enabled>
```
The same for `cache_path`